universal-application-tool-0.0.1/test/services/applicant/question/ApplicantQuestionTest.java

```diff
@@ -2,119 +2,24 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 
-import com.google.common.collect.ImmutableListMultimap;
-import com.google.common.collect.ImmutableMap;
 import com.google.common.testing.EqualsTester;
-import java.util.EnumSet;
-import java.util.Locale;
-import java.util.Optional;
 import junitparams.JUnitParamsRunner;
 import junitparams.Parameters;
-import models.LifecycleStage;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import services.Path;
 import services.applicant.ApplicantData;
 import services.question.exceptions.UnsupportedQuestionTypeException;
 import services.question.types.AddressQuestionDefinition;
-import services.question.types.CheckboxQuestionDefinition;
-import services.question.types.DropdownQuestionDefinition;
-import services.question.types.NameQuestionDefinition;
-import services.question.types.NumberQuestionDefinition;
 import services.question.types.QuestionDefinitionBuilder;
 import services.question.types.QuestionType;
-import services.question.types.RadioButtonQuestionDefinition;
 import services.question.types.TextQuestionDefinition;
 
 @RunWith(JUnitParamsRunner.class)
 public class ApplicantQuestionTest {
 
-  private static final CheckboxQuestionDefinition checkboxQuestionDefinition =
```

Comment:
(blocking) I don't think the `QuestionDefinitionBuilder.sample` is the appropriate thing to use in tests. Can you use the `TestQuestionBank`?

Reply:
Ooh yeah. I saw it being used in the `errorsPresenterExtendedForAllTypes` test, so thought that's what it was for.

I'm going to add some `public AddressQuestionDefinition addressQuestionDefinition()` (etc) to `TestQuestionBank`.

Reply:
I opened #712 last week because I think `QuestionDefinitionBuilder#sample` is weird. 

I think there already is an address question in the `TestQuestionBank` that you should be able to re-use... but maybe not.

Reply:
Chatted offline about how to use `TestQuestionBank`. Unfortunately I've gotta go for now, but I'll update this PR on Monday.

Reply:
OK, I finally fixed this. :D